keyboards/atreus/keymaps/ridingqwerty/rules.mk

```diff
@@ -0,0 +1,17 @@
+MCU = STM32F303
```

Comment:
I assume as this has f303, that you are using a proton_c? and from the LAYOUT macro, its a handwire? If that is the case, would you mind adding a `readme.md` to state that, just so we can more accurately track the exact hardware scenario.

Reply:
Correct.  I'd be happy to add a readme -- could you possibly point me to an example so I know what kind of information is desired?  Or is something like "handwired Atreus using a Proton C" sufficient?

Reply:
Yeah, what you have added is fine. Its just something so someone doesnt just refactor out the layout or MCU.